baggers.js

```diff
@@ -13207,6 +13207,80 @@ var data = {
+      "bio": "Associé chez Lucca, je suis product owner chez Lucca  depuis 2007. J'ai pu vivre le passage d'un éditeur Saas de 4 à 50 collaborateurs en 9 ans et participer à la structuration de l'organisation à chaque étape de la croissance.",
+      "picture": "https://secure.gravatar.com/avatar/4bd61d872d5e2596c631a4f1d3041451",
+      "websites": [
+        {
+          "name": "Lucca",
+          "url": "https://http://www.lucca.fr/"
+        }
+      ],
+      "location": "Nantes",
+      "sessions": [
+        {
+          "title": "Tools :  Comment construire un SI uniquement avec des outils en ligne",
+          "abstract": "Salesforce, Slack, Trello, Zendesk, Jira, Scrumdo, Teamwork, Captain train: les outils Saas envahissent notre vie professionnelle. Pourquoi est-ce inévitable ? Comment être efficace ? Comment les faire communiquer ensemble ? ",
+          "tags": [
+            "Trello",
+            "Saas",
+            "SI",
+            "Tooling",
+            "Slack"
+          ],
+          "lang": [
+            "fr"
+          ]
+        },
+        {
+          "title": "Cycle d'amélioration continue / objectifs / évaluations",
+          "abstract": "Retour d'expérience sur le suivi des objectifs collectifs et individuels chez Lucca (éditeur Saas 50 personnes) Partage du cycle d'amélioration continue à 3 niveaux (entreprise, équipe, individuel) permettant de fixer de façon trimestrielle les priorités de chaque collaborateur.",
+          "tags": [
+            "Objectifs",
+            "LeanStartup",
+            "Retour d'expérience",
+            "Evaluation"
+          ],
+          "lang": [
+            "fr"
+          ]
+        },
+        {
+          "title": "Mise en place de Scrum dans une startup",
+          "abstract": "On parle souvent de Scrum pour rendre plus agile des procédures rigides. J'ai mis scrum en place chez Lucca pour scruturer notre équipe de R&D qui n'avait aucune procédure. Si votre équipe de R&D augmente et que vous avez besoin de vous structurer pour rester efficace. Je vous partagerai les étapes que l'on a utilisées chez Lucca pour mettre en place Scrum.",
+          "tags": [
+            "Scrum",
+            "Startup",
+            "Retour d'expérience"
+          ],
+          "lang": [
+            "fr"
+          ]
+        },
+        {
+          "title": "Organiser un Startup Weekend interne",
+          "abstract": "Vous avez envie de montrer à vos équipes qu'elles peuvent résoudre des grosses problématiques en moins de 48h ? Je vous donnerai toutes les clés pour organiser un Startup Weekend en interne.",
+          "tags": [
+            "StartupWeekend",
+            "LeanStartup",
+            "Retour d'expérience",
+            "Get Things Done (in team)"
+          ],
+          "lang": [
+            "fr"
+          ]
+        }
+      ],
+      "cities": [
+        "Nantes"
+      ],
+      "contacts": {
+        "twitter": "gallain",
+        "mail": "guillaume@llain.fr"
```

Comment:
Just checking. Is your email address correct? I guess so but prefer double checking